src/log-receiver/log-receiver.service.ts

```diff
@@ -1,23 +1,28 @@
-import { Injectable, HttpService, OnModuleInit } from '@nestjs/common';
+import {
+  Injectable,
+  HttpService,
+  HttpException,
+  OnModuleInit,
+} from '@nestjs/common';
```

Comment:
in einer zeile ist schöner.
```suggestion
import { Injectable, HttpService, HttpException, OnModuleInit } from '@nestjs/common';
```